backend/typescript/models/campSession.model.ts

```diff
@@ -28,31 +33,30 @@ const CampSessionSchema: Schema = new Schema({
     ],
     default: [],
   },
-  waitlist: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "WaitlistedCamper",
-      },
-    ],
-    default: [],
-  },
   dates: {
     type: [Date],
     required: true,
   },
-  startTime: {
+  endTime: {
     type: String,
     required: true,
   },
-  endTime: {
+  priceId: {
```

Comment:
I was thinking that in `priceId`, we have `required` as false and have `default` as "". The reason being is that price Ids will be created only once `campSessions` are successfully created. So having them as required, we'll need to pass an empty string every time we are creating a campSession